src/Microsoft.Tye.Core/ApplicationBuilder.cs

```diff
@@ -13,12 +13,15 @@ public ApplicationBuilder(FileInfo source, string name)
         {
             Source = source;
             Name = name;
+            Application = name;
         }
 
         public FileInfo Source { get; set; }
 
         public string Name { get; set; }
 
+        public string? Application { get; set; }
```

Comment:
What is this for?

Reply:
From #29

> What does the name section of the manifest mean? Is that the overall application name or the name of this slice of the multi-service application?

When you "pack" your application the name is the package name, where app would be the overall application (used for tags etc)